content/en/examples/controllers/frontend.yaml

```diff
@@ -17,5 +17,5 @@ spec:
         tier: frontend
     spec:
       containers:
-      - name: php-redis
-        image: gcr.io/google_samples/gb-frontend:v3
+      - name: nginx
+        image: nginx
```

Comment:
Can you use a tag with the container image e.g. nginx:1.14.2
or any valid nginx tag
Most (besides a placeholder) use a tag with the container image in the manifest examples